system_test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -exo pipefail
+
+source ./helpers.sh
+
+remove_containers
+
+case "$1" in
+    "sqlite3" )
+    rm -fr /tmp/fn_integration_tests.db
+    touch /tmp/fn_integration_tests.db
+    export FN_DB_URL="sqlite3:///tmp/fn_integration_tests.db"
+    ;;
+
+    "mysql" )
+    DB_CONTAINER="func-mysql-test"
```

Comment:
again, the container names and ports here collide with api_test.sh, it's possible to get hit by errors here if api_test.sh is already running. For the scope of this PR, we can make sure this doesn't collide with api_test.sh, but in the long run I think we might move some of this logic to the Makefile itself where we spawn these at the beginning shared by both.